main.c

```diff
@@ -1,4 +1,98 @@
 #include <stdio.h>
+#include <stdlib.h>
+
+// Recursive function to delete a node with given key
```

Comment:
This function is copied to the geeks for geeks, right? It is valid since this project is just to training about workflow and not really to code an AVL three. But even in this project, I think we should give the credits, maybe adding a preamble in the code explaining the purpose of this repository and also where these codes are from.